src/daemon/utils_cache.c

```diff
@@ -809,7 +813,8 @@ int uc_get_hits(const data_set_t *ds, const value_list_t *vl) {
   return ret;
 } /* int uc_get_hits */
 
-int uc_set_hits(const data_set_t *ds, const value_list_t *vl, int hits) {
+int uc_set_hits(__attribute__((unused)) const data_set_t *ds,
```

Comment:
We could just remove the `ds` argument. There is only one call site.

Reply:
done